command/pr_create.go

```diff
@@ -55,6 +55,11 @@ func prCreate(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("must be on a branch named differently than %q", baseBranch)
 	}
 
+	_, err = api.PullRequestForBranch(client, headRepo, headBranch)
+	if err != fmt.Errorf("no open pull requests found for branch %q", headBranch) {
```

Comment:
I'm worried that this check will fail if we ever tweak the text of the original error message.

To reliably detect the type of an error, I think that `PullRequestForBranch()` should return a special type for the "not found" case, perhaps one named `ErrNotFound`. Then we could do this:
```suggestion
	if !errors.Is(err, api.ErrNotFound) {
```

How does this sound?

Reply:
I think, we can reuse [`api.NotFoundError`](https://github.com/cli/cli/blob/master/api/queries_pr.go#L66) here